returnn/config.py

```diff
@@ -68,10 +69,26 @@ class ReturnnConfig:
         """
     )
 
+    GET_NETWORK_CODE = textwrap.dedent(
+        """\
+        
+        def get_network(epoch, **kwargs):
+          from networks import networks_dict
+          while(True):
+            if epoch in networks_dict:
+              return networks_dict[epoch]
+            else:
+              epoch -= 1
+              assert epoch > 0, \"Error, no networks found\"
```

Comment:
I don't quite like `while(True)` loops, although in this case it might effectively be a `while(epoch>0)` loop and save. 

I suggest instead of looping over epochs and checking whether or not an associated file exists, we should loop over the existing files and compare to the epoch. With this we can save a lot of iterations, especially when epoch is much larger than `max(networks_dict.keys())`. 

```suggestion
          for epoch_ in sorted(networks_dict.keys(), reverse=True):
            if epoch_ <= epoch:
              return networks_dict[epoch_]
          assert False, \"Error, no networks found\"
```

Reply:
as a corollary, should we rather 
```assert 1 in networks_dict```
or is there a viable solution to define the initial network without providing `networks_dict[1]`?

Ah, newermind. The starting epoch could be larger than 1, then it would be fine to define only later epochs.